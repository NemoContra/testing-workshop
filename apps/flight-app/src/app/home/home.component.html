<div class="header">
  <p class="title" data-tid="home-greeting" flightColorRed>
    Welcome to Flight-App
  </p>
</div>
