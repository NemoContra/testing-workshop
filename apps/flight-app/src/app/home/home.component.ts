import { Component } from '@angular/core';

@Component({
  selector: 'flight-home',
  templateUrl: './home.component.html',
  styleUrls: ['./home.component.scss'],
})
export class HomeComponent {}
