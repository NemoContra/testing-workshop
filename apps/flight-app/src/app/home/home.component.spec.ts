import { createRoutingFactory } from '@ngneat/spectator/jest';
import { HomeComponent } from './home.component';

describe('HomeComponent', () => {
  const createComponent = createRoutingFactory({
    component: HomeComponent,
    routes: [
      {
        path: '',
        component: HomeComponent,
      },
    ],
  });

  it('should match snapshot', () => {
    const spectator = createComponent();
    expect(spectator.fixture).toMatchSnapshot();
  });
});
