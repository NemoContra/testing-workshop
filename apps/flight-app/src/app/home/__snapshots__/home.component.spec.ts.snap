// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeComponent should match snapshot 1`] = `
<flight-home
  __ngContext__="0"
>
  <div
    class="header"
  >
    <p
      class="title"
      data-tid="home-greeting"
    >
      Welcome to Flight-App
    </p>
  </div>
</flight-home>
`;
